src/main/java/nl/hsac/fitnesse/util/RandomDomain.java

```diff
@@ -3,11 +3,37 @@
 public class RandomDomain {
 
     private static final RandomUtil RANDOM_UTIL = new RandomUtil();
-    private static final String PERMITTED = "abcdefghijklmnopqrstuvwxyz1234567890-";
+    private static final String PERMITTED_NO_PERIOD = "abcdefghijklmnopqrstuvwxyz1234567890-";
+    private static final String PERMITTED_PERIOD = "abcdefghijklmnopqrstuvwxyz1234567890-.";
 
 
-    public String getRandomDomain(int length) {
-        return RANDOM_UTIL.randomString(PERMITTED, length);
+    public String getRandomSecondLevelDomain(int length) {
```

Comment:
Is it really needed to make building the 2nd level domain so complex? Or should we just generate a string with periods as possible character and just re-generate if it starts or ends with period or contains 2 periods in a row?

Reply:
Ah, that is why that branch was open.
Well, this is not ideal.
I will revisit this over the weekend and rework your comments.
While waiting, how about you review that change I made for the French Ibans?

Reply:
French IBAN looks good, maybe better in a separate branch? One thing I did consider it might be better to have long lists, like these bank codes, in a separate file (as resource) with one option per line instead of in code like this. What do you think about that?

Reply:
Yeah, nothing a little force push cannot fix.
Let's discuss the FR Iban in the other PR I will make.